### PR TITLE
124 `write.cassandra` block

### LINKED ISSUE #124: `write_cassandra` block

### PR DESCRIPTION
**Pull request by spicy-sauce**

### CODE REVIEW

File: core/src/datayoga_core/resources/schemas/connections.schema.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
             "type": {
               "description": "DB type",
               "type": "string",
-              "pattern": "^(?!redis$)"
+              "pattern": "^(?!redis|cassandra$)"
```
Comment (zalmane) [inline core/src/datayoga_core/resources/schemas/connections.schema.json:15]: what about all the other dbtypes?

File: core/src/datayoga_core/utils.py [hunk 2]
```diff
@@ -105,3 +108,47 @@ def produce_data_and_results(data: List[Dict[str, Any]]) -> Tuple[List[Dict[str,
 
 def all_success(data: List[Dict[str, Any]]) -> Tuple[List[Dict[str, Any]], List[Result]]:
     return data, [result.SUCCESS] * len(data)
+
+
+def get_field(item: Union[Dict[str, str], str]) -> str:
+    return str(next(iter(item.values()))) if isinstance(item, dict) else item
```
Comment (zalmane) [inline core/src/datayoga_core/utils.py:114]: what are these for? fields of what?

File: core/src/datayoga_core/utils.py [hunk 3]
```diff
@@ -105,3 +108,47 @@ def produce_data_and_results(data: List[Dict[str, Any]]) -> Tuple[List[Dict[str,
...
+        if key not in record:
+            logger.warning(f"{key} key does not exist in record:\n{record}")
+            raise ValueError(f"{key} key does not exist")
+
+        key_values[key] = record[key]
+
+    return key_values
+
+
+def get_field(item: Union[Dict[str, str], str]) -> str:
```
Comment (zalmane) [inline core/src/datayoga_core/utils.py:135]: isn't this a duplicate definition?

File: core/src/datayoga_core/utils.py [hunk 4]
```diff
@@ -105,3 +108,47 @@ def produce_data_and_results(data: List[Dict[str, Any]]) -> Tuple[List[Dict[str,
...
+    return str(next(iter(item.values()))) if isinstance(item, dict) else item
+
+
+def get_fields(mapping: Optional[Union[Dict[str, Any], str]]) -> List[Dict[str, Any]]:
```
Comment (zalmane) [inline core/src/datayoga_core/utils.py:139]: we already have get_fields

**Comment by spicy-sauce** [comment]
closing this one, will issue a new PR once done.

**Pull request closed without merging**